src/views/pages/events/eventdefault.js

```diff
@@ -1,12 +1,15 @@
+import React, { useState, useEffect } from 'react';
 import {
-  Box, Button, Grid, makeStyles
+Box, 
+Button, 
+Grid, 
+makeStyles 
```

Comment:
use 2 spaces indentation and Trailing spaces are not allowed  according to formator